commands/release/release.go

```diff
@@ -73,12 +73,19 @@ var (
 	`
 )
 
+var shouldBumpMajorVersion bool
 var ReleaseCmd = &cobra.Command{
 	Use:   releaseCmdStr,
 	Short: "Cuts a new release on the repo",
+	Long:  "Cuts a new release on a Kurtosis Repo. This command is intended to be ran in a github action and requires a release token to authenticate pushes to master.",
+	Args:  cobra.ExactArgs(1),
 	RunE:  run,
 }
 
+func init() {
+	ReleaseCmd.Flags().BoolVarP(&shouldBumpMajorVersion, "bump-major", "", false, "should bump the major version for this release")
```

Comment:
1) Got a bunch of magic constants in here!
2) I'd recommend being more descriptive. In Cobra this will render as:
```
bump-major   should bump the major version for this release
```
when we can be more precise:
```
bump-major      If set, in place of doing version autodetection based on the changelog, the major version ("X" in X.Y.Z) will be bumped
```
We always want to assume that the user knows quite a bit less than us (except in special cases)